components/SidebarOrder.tsx

```diff
@@ -0,0 +1,67 @@
+import { faHeart } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+
+import data from "../order.json";
+
+const ByLine = () => (
+  <div className="text-sm pt-6">
+    <h3 className="text-sm font-bold uppercase tracking-wider mb-2 text-slate">Sort By</h3>
+  </div>
+);
+
+
+export const SidebarOrder = () => {
+    const [dtOrder, setDtOrder] = React.useState(false);
+    const [starOrder, setStarOrder] = React.useState(false);
+
+    const changeDtOrder =()=>{
+        if (dtOrder === true){
+          setDtOrder(false) ;
+          data.date = false;
+    
+        }else{
```

Comment:
There should be a linter in the repo to fix these linting issues (no space between brackets and else)